app/main.py

```diff
@@ -1,2 +1,50 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.length = 0
+        self.capacity = 8
+        self.load_factor = 2 / 3
+        self.threshold = self.capacity * self.load_factor
+        self.hash_table = [[]for _ in range(self.capacity)]
+
+    def __setitem__(self, key, value):
+        if self.length > self.threshold:
+            self.capacity *= 2
+            self.threshold = self.capacity * self.load_factor
+            prev_list = [item for item in self.hash_table if item]
```

Comment:
how to create a copy of the list?

Reply:
its not copy? prev_list = [item for item in self.hash_table if item]

Reply:
yes, but we have a short and more best practice way to create a copy

Reply:
you mean i need to use deepcopy?

Reply:
but i need not copy, i don`t need empty cell

Reply:
okay, one more comment